From f0de9fd9967d4e1b4427d8458bf8983bc2cde201 Mon Sep 17 00:00:00 2001
From: Hugo Clarke-Wing <7689302+clarkewing@users.noreply.github.com>
Date: Fri, 19 Feb 2021 16:09:51 +0100
Subject: [PATCH] Don't flash 'current_password' input (#5541)

* Don't flash `current_password` input

With starter packs like Jetstream, the `current_password` input is used.

I believe that adding `current_password` to the `$dontFlash` list by default would help to ensure new projects follow security best practices from the get-go.

* Update Handler.php

Co-authored-by: Taylor Otwell <taylor@laravel.com>
---
 app/Exceptions/Handler.php | 1 +
 1 file changed, 1 insertion(+)

diff --git a/app/Exceptions/Handler.php b/app/Exceptions/Handler.php
index f9644addc0..c18c43cc12 100644
--- a/app/Exceptions/Handler.php
+++ b/app/Exceptions/Handler.php
@@ -22,6 +22,7 @@ class Handler extends ExceptionHandler
      * @var array
      */
     protected $dontFlash = [
+        'current_password',
         'password',
         'password_confirmation',
     ];
